core/src/test/scala/org/apache/spark/sql/BaseTiSparkTest.scala

```diff
@@ -377,6 +384,18 @@ class BaseTiSparkTest extends QueryTest with SharedSQLContext {
       }
     }
 
+    // test tiflash's result
+    if (spark.conf.get(TiConfigConst.ENABLE_TIFLASH_TEST).toBoolean) {
```

Comment:
make this a global variable for tests.

Reply:
Nope. TiFlash test currently is not supported by CI we have to maintain this config.

Reply:
well you don't have to read configuration from spark.conf *every time* when `runTest()` is called